src/global.css

```diff
@@ -0,0 +1,35 @@
+:root {
+  --primary-color: #00d646;
+  --secondary-color: #333b50;
+
+  --background-color-primary: #121212;
+  --background-color-secondary: #1e1e1e;
+
+  --text-color: #ffffff;
+}
+
+
+@font-face {
+  font-family: "Lulo Bold";
+  src: url(".././src/assets/Lulo\ Clean/Lulo Clean One Bold.otf") format('opentype');
```

Comment:
Make the file names as small and clear as possible

`.././src/assets/Lulo\ Clean/Lulo Clean One Bold.otf` 👉 `.././src/assets/lulo-clean/lulo-clean-bold.otf`